### PR TITLE
Python: Updated and add tests for Hardcoded Passwords

### PR DESCRIPTION
**Pull request by GeekMasher**

### CODE REVIEW

File: tests/python-tests/CWE-798/hardcoded_secrets.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+from mysql.connector import connect
+
+conn = connect(user="user", password="mysecretpassword")
+
+# Asyncpg
+from asyncpg import connect
+from asyncpg.connection import Connection
+
+asyncpg_conn1 = await connect(user="user", password="asyncpg_secret1")
+asyncpg_conn2 = Connection(user="user", password="asyncpg_secret2")
```
Comment (GeekMasher) [inline tests/python-tests/CWE-798/hardcoded_secrets.py:35]: Missed

File: tests/python-tests/CWE-798/hardcoded_secrets.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import os
+
+password = os.environ.get("SECRET_TOKEN")
+
+# Flask
+from flask import Flask
+
+app = Flask(__name__)
+
+app.secret_key = "SecretKey1"
```
Comment (GeekMasher) [inline tests/python-tests/CWE-798/hardcoded_secrets.py:10]: Missed

File: tests/python-tests/CWE-798/hardcoded_secrets.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+
+# Django
+SECRET_KEY = "SuperSecretKey"  # False Positive, not a settings file
+
+
+# Requests
+from requests.auth import HTTPBasicAuth
+
+auth = HTTPBasicAuth("user", "mysecretpassword")
```
Comment (GeekMasher) [inline tests/python-tests/CWE-798/hardcoded_secrets.py:22]: Missed

File: tests/python-tests/CWE-798/hardcoded_secrets.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+
+# MySQL
+from mysql.connector import connect
+
+conn = connect(user="user", password="mysecretpassword")
```
Comment (GeekMasher) [inline tests/python-tests/CWE-798/hardcoded_secrets.py:28]: Missed

**Review by securingdev — APPROVED** [review]
LGTM.

**Comment by GeekMasher** [comment]
@securingdev Waiting for #48 to be merged as the tests are failing because of some of the API changes in CodeQL

**Comment by aegilops** [comment]
Can you resolve the merge conflicts and then merge this please?

**Pull request closed without merging**